intelmq/tests/test_conf.py

```diff
@@ -50,13 +50,6 @@ class TestConf(unittest.TestCase):
     A TestCase for configuration files.
     """
 
-    def test_defaults_syntax(self):
-        """ Test if defaults.conf has correct syntax. """
-        with open(CONF_FILES['defaults']) as fhandle:
-            fcontent = fhandle.read()
-        interpreted = json.loads(fcontent)
-        self.assertEqual(to_json(interpreted), fcontent)
-
```

Comment:
Why do you remove the test if the file still exist?

Reply:
Ah, because I tried to get rid of the file, but there is other code still depending on it (i.e. in `intelmqctl`)- I forgot to readd the test. Fixed now.